Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1105,8 +1105,33 @@ int Element::scroll_width() const
 
```

Comment:
Could you add a spec link? (https://w3c.github.io/csswg-drafts/cssom-view/#dom-element-scrollheight I think)

Reply:
Sure

Reply:
Thanks!